src/tools/demo.ts

```diff
@@ -0,0 +1,57 @@
+export enum CommentType {
+  REMOVE_FILE = `@demo remove-file`,
+  REMOVE_CURRENT_LINE = `@demo remove-current-line`,
+  REMOVE_NEXT_LINE = `@demo remove-next-line`,
+  REMOVE_BLOCK_START = `@demo remove-block-start`,
+  REMOVE_BLOCK_END = `@demo remove-block-end`,
```

Comment:
So in order to support `//`, `/* */`, and `{/* */}` jsx comments, I decided to just include the `@demo remove-*` string as the keywords.

This could potentially run into errors where we find the keywords as strings and remove those lines instead of comments. However, I can't think of a use case where we'd write these keywords and not have them in a comment.

The alternative is bringing in much more heavy tooling with using an AST to look for comment tokens. That may be worth it, but I think this simpler approach of find and replace is worth it for now

Reply:
I'm pretty against going to an AST, just because the understandability goes wayyy down and you'll be maintaining that yourself until you die. This simpler (less powerful) approach is superior.

Reply:
I'm in the same boat, I think for this use case find and replace is perfect